### PR TITLE
Stereoviewer

### PR DESCRIPTION
**Pull request by gunrot**

### CODE REVIEW

File: transitions/StereoViewer.glsl [hunk 1]
```diff
@@ -0,0 +1,210 @@
+// Tunable parameters
+// How much to zoom (out) for the effect ~ 0.5 - 1.0
+uniform float zoom; // = 0.88
+// Corner radius as a fraction of the image height
+uniform float corner_radius;  // = 0.22
+
+// author: Ted Schundler
+// license: BSD 2 clause
```
Comment (gltransitions) [inline transitions/StereoViewer.glsl]: 'BSD 2 clause' not found in supported licenses: MIT, EPL-1.0, CDDL-1.0, MPL-2.0, LGPL, GPLv2, GPLv3, Apache-2.0

Comment (gunrot) [reply transitions/StereoViewer.glsl]: But the license is BSD 2 clause. It is not my choice.

Comment (gre) [reply transitions/StereoViewer.glsl]: ok, hold on, i'm going to had it in the whitelist

File: transitions/StereoViewer.glsl [hunk 2]
```diff
@@ -0,0 +1,210 @@
+// Tunable parameters
+// How much to zoom (out) for the effect ~ 0.5 - 1.0
+uniform float zoom; // = 0.88
+// Corner radius as a fraction of the image height
+uniform float corner_radius;  // = 0.22
+
+// author: Ted Schundler
+// license: BSD 2 Clause
```
Comment (gltransitions) [inline transitions/StereoViewer.glsl]: 'BSD 2 Clause' not found in supported licenses: MIT, EPL-1.0, CDDL-1.0, MPL-2.0, LGPL, GPLv2, GPLv3, Apache-2.0

Comment (gltransitions) [inline transitions/StereoViewer.glsl]: 'BSD 2 Clause' not found in supported licenses: MIT, EPL-1.0, CDDL-1.0, MPL-2.0, LGPL, GPLv2, GPLv3, Apache-2.0

Comment (gltransitions) [inline transitions/StereoViewer.glsl]: 'BSD 2 Clause' not found in supported licenses: MIT, EPL-1.0, CDDL-1.0, MPL-2.0, LGPL, GPLv2, GPLv3, Apache-2.0

**Comment by gunrot** [comment]
What is the problem? I have 3 more transitions in my fork, but first want to get experience with this one.

**Comment by gre** [comment]
just a problem with my bot redeployment^^

**Comment by gre** [comment]
omg is that it? I think I finally figured out my bot deployment fix. 😅coming back to a docker instance 1 year later and everything is broken (`node:latest` 😅)

**Comment by gre** [comment]
there seem to be a problem in the transition as shown in the gif?

**Comment by gunrot** [comment]
Yes I made a silly mistake:
vec4 simple_sample_with_corners_to(vec2 p, vec2 corner_size, float zoom_amt) {
+  p = (p - 0.5) / (1.0 - zoom_amt + zoom * zoom_amt) + 0.5;
+  if (!test_rounded_mask(p, corner_size)) {
+    return black;
+  }
+  return getFromColor(p);
+}

getFromColor must be getToColor.

**Comment by gre** [comment]
can you edit it?

**Comment by gunrot** [comment]
Done.

**Comment by gre** [comment]
Cool!

**Pull request merged**